chapters/introduction.tex

```diff
@@ -1,11 +1,7 @@
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren.
+With technology steadily improving over the years, people are becoming more interconnected than ever and entertainment in form of applications are plentiful. Despite this, depression continues to rise among the general population. According to the World Health Organization, over 322 million cases of depressive disorder were recorded in 2017, representing a rise of around 18.4\% over the past ten years \cite{who_depression}. Not only does depression negatively impact wellbeing, it may also lead to complications such as increased fatigue, decreased motivation or even suicide. As a result, researchers have used computing technologies known as \textit{Affective Computing} that seek to identify human emotions in order to combat mental disorders \cite{ieee_affective}.
 
-The research objective of this paper is to design and implement the approach proposed by Moore, Galway and Donnelly in \cite{sohappy}. Finally, a conclusion is drawn in section \ref{sec:conclusion}.
+In \cite{sohappy}, Moore, Galway and Donnelly propose a smartphone application to be used in a future study. It is designed to encourage smiling, which has been shown to positively affect happiness and thus serves as an effective means to counteract depression. The research objective of this report is to design and implement the approach described by Moore, Galway and Donnelly as an Android application. In order to render the application suitable for different kinds of studies, the application's components are designed to be as interchangeable as possible, allowing individual adjustments to be made with ease.
 
-% This paper is organized as follows. An introduction to different use cases and priority strategies is given in Section \ref{sec:usecases}. In Section \ref{sec:solution}, the advantages and drawbacks of two different solutions are discussed. Finally, we conclude in Section \ref{sec:conclusion}.
+The report starts with an overview of appropriate approaches to implement the application in Section \ref{sec:background}. In Section \ref{sec:methodology}, the design choices and architecture of the app are described in detail, followed by a description of the implementation details in Section \ref{sec:implementation}. After presenting the development outcome in Section \ref{sec:results} and a discussion about alternative approaches in Section \ref{sec:discussion}, Section {sec:conclusion} concludes the report with ideas for further work on the app.

```

Comment:
Is this part replay necessary? Aren't we going to include a ToC?

Typo here: {sec:conclusion} - missing \ref

Reply:
This part is slightly distinct from a TOC in that it describes what kind of information each segment contains. But you're right, I don't think it's completely necessary.

Reply:
Based on yesterday's meeting with Leo, we should probably leave this part as is.